EIPS/eip-3643.md

```diff
@@ -363,19 +368,46 @@ interface IClaimTopicsRegistry {
+
+### Identity Management
+
+Security and compliance of transfers are enforced through the management of on-chain identities. These include:
+
+- Identity contract: A unique identifier for each investor, which is used to manage their identity and claims.
+- Claim: Signed attestations issued by a trusted claim issuer that confirm certain attributes or qualifications of the token holders, such as their identity, location, investor status, or KYC/AML clearance.
+- Identity Storage/Registry: A storage system for all Identity contracts and their associated wallets, which is used to 
+  verify the eligibility of investors during transfers.
+
+### Token Lifecycle Management
+
+The T-REX standard provides a comprehensive framework for managing the lifecycle of security tokens. This includes the issuance of tokens, transfers between eligible investors, and the enforcement of compliance rules at every stage of the token's lifecycle. The standard also supports additional features such as token pausing and freezing, which can be used to manage the token in response to regulatory requirements or changes in the status of the token or its holders.
+
+### Additional Compliance Rules
+
+The T-REX standard supports the implementation of additional compliance rules through modular compliance. These modules can be used to enforce a wide range of rules and restrictions, such as caps on the number of investors or the percentage of tokens held by a single investor, restrictions on transfers between certain types of investors, and more. This flexibility allows issuers to tailor the compliance rules of their tokens to their specific needs and regulatory environment.
+
+## Backwards Compatibility
+
+T-REX tokens should be backwards compatible with ERC-20 and ERC-173
+and should be able to interact with a [Claim Holder contract](../assets/eip-3643/ONCHAINID/IERC735.sol) to validate 
+the claims linked to an [Identity contract](../assets/eip-3643/ONCHAINID/IIdentity.sol).
+
 ## Test Cases
 
-The standard is implemented and tested with full coverage on Tokeny's [T-REX repository](https://github.com/TokenySolutions/T-REX)
+The standard is implemented and tested with full coverage on Tokeny's T-REX repository.
 
 ## Security Considerations
 
-The suite of Smart Contracts has been audited by an external and independent company. The results can be found in [this document](https://tokeny.com/wp-content/uploads/2020/05/Tokeny-Solutions_T-REX-v3_Smart-Contract-Audit-Report_Kapersky.pdf).
+The suite of Smart Contracts has been audited by external independent companies. The results can be found on 
+Tokeny's website or on the Tokeny's T-REX repository.
```

Comment:
Mentioning your website is conceptually an "external link", and so isn't allowed. I'd recommend either summarizing the results of the audit, or simply noting that no security issues have been found.

Perhaps something like one of these, as appropriate:

---

> This specification has been audited by Kapersky, and no notable security considerations were found.

---

> An audit, performed by Kapersky, highlighted the following considerations for implementers of this standard:
>  * When flooping a glarp, implementers must pay particular attention to the meep. It must remain glooped for the duration of the merp.
>  * Never transfer the moop to the merp. Doing so would allow an attacker to floop the glarp too early.

---

From an _extremely_ quick read of the audit report, it looks like it was focused on your implementation, but **not on this standard itself**. In other words, they focused on whether your implementation conformed to your whitepaper, and didn't consider whether your overall design was sound.

That isn't problematic in and of itself, but it does mean the audit might not be applicable to other implementers?

Reply:
Indeed, the audit mainly focused on the concrete implementation of the standard. However, it is important to note that our implementation is a direct translation of the standard specification outlined in the whitepaper. Therefore, the audit process essentially included a thorough review and verification of the standard's design and functionality.

While the audit report may not explicitly state the details of these discussions, the audit process was comprehensive and spanned over four months. During this time, the audit team harshly challenged the principles and design decisions underlying the T-REX standard (ERC-3643). Their inquiry was not limited to the security aspects of smart contracts, but also delved into the rationale behind the standard's specifications. Satisfactory responses to these inquiries and the absence of any outstanding safety issues in the audit report demonstrate the robustness of the standard.

It is true that the audits we realized may not be fully applicable to other implementations of the standard, but still provide valuable insight into the standard's design and functionality. The audits validate the standard's specifications and provide assurance that the standard does not present any notable security issues (when implemented as per the specifications). Therefore, we consider it relevant and useful to mention the audits in this section related to standard security considerations. 

Reply:
Updated the `Security Considerations` section in https://github.com/ethereum/EIPs/pull/7164/commits/adfc130d6dd1ed41fd86709401c7ed93b8d6178c